### DISCUSSION TITLE
any update on nuking 3kh0's facilities/residence

### DISCUSSION THREAD

**Discussion by Theblackmamba824** [3kh0/ext-remover#326] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
how is the nuke going we dont got all day

**Comment by rwg100** [comment, marked as answer]
well SOMEONE has to enrich the uranium and guess who picked the short straw!

**Reply by Aka-but-nice** [reply]
like how is that not obvious that's light work, I learned that in first grade. Simple stuff.

**Reply by 3kh0** [reply]
Buy it off of some dark net market place on tor, like the silk road or something
